src/storage/mod.rs

```diff
@@ -485,7 +485,9 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
                         range.0.as_ref().map(AsRef::as_ref),
                         range.1.as_ref().map(AsRef::as_ref),
                     );
-                    if ApiV2::parse_range_mode(range) != KeyMode::TiDB {
+                    if ApiV2::parse_range_mode(range) != KeyMode::TiDB
```

Comment:
Suggest to just remove `check_api_version_ranges` in `scan_lock` [here](https://github.com/tikv/tikv/blob/1fb8980ccab9ff40c1adc206df52952dab8e8ad8/src/storage/mod.rs#L1267). Also leave comments about why.

Reply:
OK. I already fix it in ```scan_lock```, and use the old conditional judgment inside ```check_api_version_ranges```.